src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working.");

```

Comment:
The message should indicate what type of `Machine` is working right now. Consider specifying that it is a `Bulldozer` to make the message more informative.